src/Backend/Modules/Blog/Domain/Category/CategoryRepository.php

```diff
@@ -7,7 +7,13 @@
 
 class CategoryRepository extends EntityRepository
 {
-    public function getUrl(string $url, ?int $id)
+    public function remove(Category $category): void
+    {
+        $this->getEntityManager()->remove($category);
+        $this->getEntityManager()->flush($category);
+    }
+
+    public function getUrl(string $url, ?int $id): string
```

Comment:
should be `int $id = null` since you only need the id when you want to update an existing url and otherwise you'll have to pass null when you actually don't need the second parameter

Reply:
Fixed in ae614a62d315d074c1532c1f207abb2bd14d4a99